subprojects/language-native/src/integTest/groovy/org/gradle/language/cpp/CppLanguageIncrementalBuildIntegrationTest.groovy

```diff
@@ -16,10 +16,16 @@
 
 package org.gradle.language.cpp
 
+import org.gradle.integtests.fixtures.ToBeFixedForVfsRetention
 import org.gradle.language.AbstractNativeLanguageIncrementalBuildIntegrationTest
 import org.gradle.nativeplatform.fixtures.app.CppHelloWorldApp
 import org.gradle.nativeplatform.fixtures.app.IncrementalHelloWorldApp
+import org.gradle.util.TestPrecondition
 
+@ToBeFixedForVfsRetention(
+    because = "https://github.com/gradle/gradle/issues/12182",
```

Comment:
I updated this link to point to https://github.com/gradle/gradle/issues/12162, given that https://github.com/gradle/gradle/issues/12182 is a duplicate.